### DISCUSSION TITLE
Question about to train models

### DISCUSSION THREAD

**Discussion by Victor1890** [vladmandic/human#461] [Q&A]
Hi @vladmandic, I'm playing with human and this project it's fantastic but, I want to know how can I train face model?? or Which one the project you use for train the models? or It is possible use human for train the models?

Or, Can you do one tutorial about how to train any models??

thanks!!!

**Comment by vladmandic** [comment]
what do you mean by training the face model? face detection or face mesh?
you can use pretty much any face detection model if you swap provided ones.
facemesh is pretty much what it is, it cannot be easily retrained. and source comes from google mediapipe - that is all in the notes (as credits with links for every model used).

**Reply by Victor1890** [reply]
In this case I'm talking about the face detection, I want to know if you use another's tools for train the model detection.
